### DISCUSSION TITLE
course/04

### DISCUSSION THREAD

**Discussion by giscus** [mitre/inspec-advanced-developer#7] [Ideas]
# course/04

The MITRE Security Automation Framework Team's Advanced InSpec profile development course

https://mitre-inspec-advanced-developer.netlify.app/course/04.html

**Comment by karikarshivani** [comment]
A suggestion for the validate threshold output, maybe it shouldn’t say `Error` if the command executed successfully. It could say `Failure` because the user’s report didn’t meet requirement but saf validate successfully evaluated the report. `Error` should be shown in the output only when saf validate is not able to run. Seeing an output with the word `Error` in it makes me think that something went wrong with the command and I need to modify and re-run the command.
